src/MessageContent.ts

```diff
@@ -57,6 +57,7 @@ export interface ContentCodec<T> {
   contentType: ContentTypeId
   encode(content: T, registry: CodecRegistry): EncodedContent
   decode(content: EncodedContent, registry: CodecRegistry): T
+  fallback(content: T): string | undefined
```

Comment:
Love this. 

I think this qualifies as a breaking change, since all existing codecs will no longer satisfy the type.

Reply:
I don't understand how this is meant to work. Presumably the client that needs the fallback does not have the codec, otherwise it can decode the content instead. What am I missing?

Reply:
nm, I see now, this is invoked during encoding, not decoding.